### ISSUE TITLE
The training details on Clothing1M

### ISSUE THREAD

**Issue by youjiangxu** [takuhirok/rGAN#3]
Hi @takuhirok ,

     I find that there are three different settings (50K clean, 1M noisy, mixed) on the Clothing1M. Also, Clothing1M has 50K clean training dataset, 14K validation dataset, and 10K test dataset.
     I am confused about the results of Tab.5 in the paper, _i.e.,_  can you tell me that the model is trained on the 1M training data and then tested on these three settings? or the model is trained on the 50K clean dataset and tested on the 10 test dataset? 

best,
Youjiang

**Comment by takuhirok** [comment]
We trained the models on three settings: (1) 50k clean data, (2) 1M noisy data, and (3) mixed data.

In evaluation, we calculated the FID between the 50k generated samples and all the samples in the training set
(particularly, we used 1M noisy data).
Regarding the GAN-train, we trained the classifier using 50k generated samples. Using this classifier, we calculated the accuracy for a test set (particularly, we used 10k test data).
